packages/components/tests/dummy/app/templates/components/link-to/cta.hbs

```diff
@@ -0,0 +1,292 @@
+{{page-title "LinkTo::CTA (Call To Action)"}}
+
+<h2 class="dummy-h2">LinkTo::CTA (Call To Action)</h2>
+
+<section>
+  <h3 class="dummy-h3">
+    Overview
+  </h3>
+
+  <p class="dummy-paragraph">The LinkTo::CTA component handles the generation of an
+    <a
+      href="https://guides.emberjs.com/release/routing/linking-between-routes/#toc_the-linkto--component"
+      target="_blank"
+      rel="noopener noreferrer"
+    >Ember LinkTo</a>
+    component with the same exact visual appearence as the "primary"
+    <LinkTo @route="components.button">Button</LinkTo>
+    component, that points to an internal application link or resource.
+  </p>
+  <p class="dummy-paragraph">
```

Comment:
I think we also should say that there should only be one CTA on any given page.

Reply:
@heatherlarsen can you chime on this specific point? thanks

Reply:
I agree that it is too restrictive and hard to enforce, but I do think we can caution against over use in the docs coupled with education on the difference between Buttons, CTAs, and Links. 

@ashleemboyer also had some really good points [in the RFC](https://docs.google.com/document/d/1SqsHvv7_Kg5mu9FXdVS1Y7Dw4fbxXYLddNebd4Kd1ic/edit?disco=AAAAU-E6sYc) about how CTAs get used in our marketing sites which feels important to consider here as well.

Reply:
@heatherlarsen once approved and merged this PR, I'll open a follow-up issue to update the documentation for  Link(To)::CTA, Link(To)::Standalone and Button components, to explain the differences between them, and guide the users in choosing which one to use for different contexts (see first TODO in the PR description) 
That can be a good place where iron out exactly what we want to say to the users/consumers.